lib/userConf.js

```diff
@@ -69,11 +69,34 @@ const getConf = async function getConf(key, envVar, ...defaultValue) {
   return userConf[key];
 };
 
+// Matches printable ASCII-only values.
+const ascii = /^[\u0020-\u007E]*$/;
```

Comment:
empty strings should probably be disallowed too

Reply:
*Not* done.
1. This is _not_ general enforcement of having nice values, this is specific enforcement that these values can easily be transferred in HTTP headers without the request client complaining.
2. An empty string is a perfectly good string.  Why enforce it on the client?